### ISSUE TITLE
transfer ownership to Gratipay account

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay-everywhere#33]
Right now the Chrome app is hosted under @clone1018's personal Google account. We should move it under a dedicated Gratipay account. Here's the form to fill out:

https://support.google.com/chrome_webstore/contact/dev_account_transfer

**Comment by kzisme** [comment]
@whit537  Have you gotten a chance to look into doing this? I feel as though this has been overlooked for awhile, and @clone1018 haven't chimed in lately.

**Comment by chadwhitacre** [comment]
Should we use support@gratipay.com for this, or a dedicated account to reduce security surface area if support@ were to be compromised?

**Comment by chadwhitacre** [comment]
In either case we should have @clone1018 fill out [the form](https://support.google.com/chrome_webstore/contact/dev_account_transfer) since it asks for original author email and original order number.

**Comment by clone1018** [comment]
Just let me know where to point the ship

**Comment by chadwhitacre** [comment]
@clone1018 Go ahead and use vendors@gratipay.com. That's what we generally use for vendors, unless the email is going to be public, in which case we'd use support@.

**Comment by clone1018** [comment]
"Thanks for contacting us. We'll follow up with you only if we need more information or have additional information to share."

**Comment by chadwhitacre** [comment]
!m @clone1018

**Comment by clone1018** [comment]
```
The destination account must be registered in order to complete the account ownership transfer. See more information in the link below (Step 9) on how to complete the registration:

https://developers.google.com/chrome/web-store/docs/get_started_simple

Once the registration for the destination account is complete, please reply to this email so we can proceed with the transfer.
```

**Comment by chadwhitacre** [comment]
I had to enable Google Wallet for our org within Google Apps. Once I did that I was able to complete the purchase:

![screen shot 2015-07-20 at 6 58 52 pm](https://cloud.githubusercontent.com/assets/134455/8789537/67460e40-2f11-11e5-9b06-e0a1d8fca6c4.png)

Good to go, @clone1018! :-)

**Comment by chadwhitacre** [comment]
(P.S. I turned Wallet back off.)

**Comment by chadwhitacre** [comment]
![screen shot 2015-07-20 at 7 00 59 pm](https://cloud.githubusercontent.com/assets/134455/8789557/ae913112-2f11-11e5-8e51-60812fe39306.png)

**Comment by nobodxbodon** [comment]
As we are discussing about [reviving the extensions](https://github.com/gratipay/inside.gratipay.com/issues/893#issuecomment-261826429), shall we continue here? Plus the [firefox extension](https://addons.mozilla.org/zh-CN/firefox/addon/gittip/?src=search)

**Comment by chadwhitacre** [comment]
I just reread this ticket, and I think the 🏀  is in @clone1018's court?

**Comment by clone1018** [comment]
> Thanks for contacting us. We'll follow up with you only if we need more information or have additional information to share.

**Comment by clone1018** [comment]
I'm assuming the first submission will be rejected. It asks for an order number and I have a "Google Order Number" and a "Chrome Webstore Order Number"

**Comment by clone1018** [comment]
> Dear Luke,
>
> Thank you for your approval email. We can confirm that the ownership was transferred as requested.
>
> Sincerely,
> Chrome Web Store Team

**Comment by chadwhitacre** [comment]
Huzzah! So are we done here? @nobodxbodon Do you have what you need?

**Comment by nobodxbodon** [comment]
@whit537 I think so. Create another issue to do the same for [Firefox extension](https://addons.mozilla.org/en-US/firefox/addon/gittip/)?

**Comment by chadwhitacre** [comment]
> Create another issue to do the same for Firefox extension?

Sure, if need be.
